iane056/technical-interview-webapp
# Spaced Repetition Model

This will be the base of our SR model. This currently assumes that the unsupervised model will spit out a value from 0 - 5.

To test the code:
- Just run the python file, either via VSCode's CodeRunner or by doing `python3 ai/spaced-repetition/main.py`
- You can also edit the qualities to see if the spaced out days seem appropriate.

To test logic:
- The main thing that needs revision is the `update()` function. Make sure it follows this [logic flow](https://www.supermemo.com/en/blog/application-of-a-computer-to-improve-the-results-obtained-in-working-with-the-supermemo-method).
